visual-testing-app/src/test-utils/spec.tsx

```diff
@@ -1,6 +1,6 @@
 import { Children, isValidElement, ReactElement, ReactNode } from 'react';
 import styled from '@emotion/styled';
-import { customProperties } from '@commercetools-uikit/design-system';
+import { designTokens } from '@commercetools-uikit/design-system';
```

Comment:
nit: for consistency
```suggestion
import { designTokens as uiKitDesignTokens } from '@commercetools-uikit/design-system';
```

Reply:
Updated here: 83560abf88251355859415b1b7577d32f0ed8b41